README.md

```diff
@@ -1,30 +1,30 @@
 # PyTorch Benchmarks
 This is a collection of open source benchmarks used to evaluate PyTorch performance.
 
-`torchbenchmark/models` contains copies of popular or exemplary workloads which have been modified to
-(a) expose a standardized API for benchmark drivers, (b) optionally, enable JIT,
- (c) contain a miniature version of train/test data and a dependency install script.
+`torchbenchmark/models` contains copies of popular or exemplary workloads which have been modified to:
+*(a)* expose a standardized API for benchmark drivers, *(b)* optionally, enable JIT,
+ *(c)* contain a miniature version of train/test data and a dependency install script.
 
 ## Installation
 The benchmark suite should be self contained in terms of dependencies,
 except for the torch products which are intended to be installed separately so
 different torch versions can be benchmarked.
 
 ### Using Pre-built Packages
-We support python 3.7+, and 3.8 is recommended. Conda is optional but suggested. To start with python 3.8 in conda:
+We support Python 3.7+, and 3.8 is recommended. Conda is optional but suggested. To start with Python 3.8 in conda:
 ```
-# using your current conda environment:
+# Using your current conda environment:
 conda install -y python=3.8
 
-# or, using a new conda environment
+# Or, using a new conda environment:
 conda create -n torchbenchmark python=3.8
 conda activate torchbenchmark
-# we depend on git lfs tool to store minimal input dataset such as images and annotations
-# the total size of input dataset is ~20 MB
+# We depend on git lfs tool to store minimal input dataset such as images and annotations.
+# The total size of input dataset is ~20 MB
 conda install -y git-lfs
 ```
 
-If you are running Nvidia GPU tests, we support CUDA 10.2+, and CUDA 11.3 is recommended:
+If you are running NVIDIA GPU tests, we support CUDA 10.2+, and CUDA 11.3 is recommended:
```

Comment:
We now support CUDA 11.3+ and use CUDA 11.6 as default.

Reply:
Just to clarify, do we not support CUDA 10.2 then, only 11.3+? So wording changes to:
> If you are running NVIDIA GPU tests, we support CUDA 11.3+, and use CUDA 11.6 as default:

This will also need to change then?
```conda install -y -c pytorch magma-cuda113```
to
```conda install -y -c pytorch magma-cuda116```

Reply:
Yes, let's change the wording to support CUDA 11.3+, and use CUDA 11.6 as default.

Yes, we also need to change the magma version to `magma-cuda116`.